### ISSUE TITLE
Custom config.js requires declaring in your application.js file

### ISSUE THREAD

**Issue by tbrammar** [SammyLin/redactor-rails#141]
You may want to add this to the read me:

If you use a custom config.js if `app\assets\redactor-rails\config.js` you must declare it in your `application.js` for it to be picked up and precompiled as part of the asset pipeline.

**Comment by tbrammar** [comment]
Ahh, I see @luaysuarna has picked this up in a pull request from a few day ago: c498e3ca549a4326127f0a10e031be1899486625

**Comment by luaysuarna** [comment]
Yes, Just simple config..I found issue in my apps, my custom config.js not precompile on staging.
I tried to change require on application.js, and Worked.
